From 29b21b09d43955f20c75a5f09cc5455e2c9fafcc Mon Sep 17 00:00:00 2001
From: Christian Boltz <apparmor@cboltz.de>
Date: Tue, 6 Jun 2023 23:29:14 +0200
Subject: [PATCH] Fix invalid aa-status --json

The previous patch changed the final  }}  to  }  - which is correct in
master, but breaks the code in the 3.x branches.
---
 binutils/aa_status.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/binutils/aa_status.c b/binutils/aa_status.c
index 40a854beb..092bee55b 100644
--- a/binutils/aa_status.c
+++ b/binutils/aa_status.c
@@ -548,7 +548,7 @@ static int detailed_output(FILE *json) {
 		if (need_finish > 0) {
 			fprintf(json, "]");
 		}
-		fprintf(json, "}\n");
+		fprintf(json, "}}\n");
 	}
 
 exit:
-- 
GitLab

